testcafe/tests/drive/search.js

```diff
@@ -0,0 +1,108 @@
+import { driveUser } from '../helpers/roles'
+//import { Role } from 'testcafe'
+import { TESTCAFE_DRIVE_URL, SLUG } from '../helpers/utils'
+import { initVR } from '../helpers/visualreview-utils'
+import { FOLDER_NAME, maskDriveFolderWithDate } from '../helpers/data'
+import PrivateDrivePage from '../pages/drive/drive-model-private'
+//import ViewerDrivePage from '../pages/drive-viewer/drive-viewer-model'
+
+const privateDrivePage = new PrivateDrivePage()
+//const viewerDrivePage = new ViewerDrivePage()
+
+//Scenario const
+const FEATURE_PREFIX = 'Search'
+const TEST_SEARCH1 = `1-1 Search letters (files)`
+const TEST_SEARCH2 = `1-2 Search letters (folder)`
+const TEST_SEARCH3 = `1-3 Search inside folder`
+const TEST_SEARCH_NO_RESULT = `1-4 Search no result`
+
+//************************
+//Tests when authentified
+//************************
+fixture`${FEATURE_PREFIX}`.page`${TESTCAFE_DRIVE_URL}/`
+  .before(async ctx => {
+    await initVR(ctx, SLUG, FEATURE_PREFIX)
+  })
+  .beforeEach(async t => {
+    console.group(`\n↳ ℹ️  Login & Initialization`)
+    await t.useRole(driveUser)
+    await privateDrivePage.waitForLoading()
+    console.groupEnd()
+  })
+  .after(async ctx => {
+    await ctx.vr.checkRunStatus()
+  })
+
+test(TEST_SEARCH1, async () => {
+  console.group(`↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_SEARCH1}`)
+  await privateDrivePage.typeInSearchInput({
+    text: 'a',
+    screenshotPath: `${FEATURE_PREFIX}/${TEST_SEARCH1}-A`
+  })
+
+  await privateDrivePage.typeInSearchInput({
+    text: 't',
+    screenshotPath: `${FEATURE_PREFIX}/${TEST_SEARCH1}-T`
+  })
+  await privateDrivePage.openSearchResultByIndex(0)
+  // For now, there is a bug when opening the viewer from search. So waitForLoading and screenshot are commented to avoid unstable tests
```

Comment:
Is that bug referenced somewhere?

Reply:
Thomas talk about it in the trello card for this PR. 
I reproduce it while testing this test. I'll add a trello link in code if there is a card for it (did find it for now, waiting for thomas to answer on mattermost! ;) )

Reply:
https://trello.com/c/N8kFROf3